<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutCheque</class>
 <widget class="QDialog" name="AjoutCheque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AjoutCheque</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>No de compte :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Taux d'interet :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Solde :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nbre Transactions :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Taux min :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAjoutCheque">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_compte">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tauxInteret">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Solde">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Description">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Transactions">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tauxmin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonAjoutCheque</sender>
   <signal>clicked()</signal>
   <receiver>AjoutCheque</receiver>
   <slot>slotConfirmerCheque()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotConfirmerCheque()</slot>
 </slots>
</ui>
